tests/support/component.json

```diff
@@ -10,8 +10,7 @@
     "../../node_modules/superagent/lib/utils.js"
   ],
   "main": "../../node_modules/superagent/lib/client.js",
-  "dependencies": {
-    "component/emitter": "*",
-    "component/reduce": "*"
```

Comment:
This was used (I'm guessing) to overcome the aliasing superagent did on their code.
Once superagent removed the use of component-reduce and stopped aliasing component-emitter (which broke the unit tests by the way) I debugged the module-as-component flow of code and realized the dependencies part of the config doesn't contribute anymore to the process, making it obsolete.
And another thing, If we are moving to use Jest (as I see from an upcoming pull request) this will not matter.